### PR TITLE
fix: trial comparison text overflow bug fix

### PR DESCRIPTION
**Pull request by keita-determined**
## Description
[WEB-1994]

### Before

see screenshot in the ticket

### After

image

## Test Plan
- Check if long text doesnt overflow in trail comparison view
- Check if all values (string, number, boolean) display correctly
- Check if tooltip is shown by hover when text is truncated
- Check if sticky cells (Metrics and Hyperparameter dropdowns) works when scroll horizontally

## Checklist

- [x] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

## Ticket

[WEB-1994]: https://hpe-aiatscale.atlassian.net/browse/WEB-1994?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by EmilyBonar — COMMENTED** [review]
This looks like it works, but I think it'd be better to use Typography instead so that there's a tooltip with the full value.

**Comment by keita-determined** [comment]
> This looks like it works, but I think it'd be better to use Typography instead so that there's a tooltip with the full value.

these text are wrapped by `Label` from `import { Label } from 'hew/Typography';`, so there's a tooltip on it

**Comment by EmilyBonar** [comment]
>> This looks like it works, but I think it'd be better to use Typography instead so that there's a tooltip with the full value.
>
> these text are wrapped by `Label` from `import { Label } from 'hew/Typography';`, so there's a tooltip on it

I don't think that's true. The relevant section of code is
```
{selectedMetrics.map((metric) => (
                <tr key={`${metric.group}-${metric.name}`}>
                  <th scope="row">
                    <MetricBadgeTag metric={metric} />
                  </th>
                  {trialsDetails.map((trial) => {
                    const metricValue = latestMetrics[trial.id][metric.name];
                    return (
                      <td key={trial.id}>
                        {metricValue !== undefined ? (
                          typeof metricValue === 'number' ? (
                            <HumanReadableNumber num={metricValue} />
                          ) : (
                            metricValue
                          )
                        ) : null}
                      </td>
                    );
                  })}
                </tr>
              ))}
```
`HumanReadableNumber` includes truncation and a tooltip, so that's fine, but if `metricValue` is a string (or boolean, though those won't overflow) it just gets plopped down into the DOM.

Also, I don't know how I managed to edit your comment instead of replying - whoops!

**Comment by keita-determined** [comment]
ah you are right. wrapped `metricValue` with `Label`

**Comment by keita-determined** [comment]
@EmilyBonar could you review this?

File: webui/react/src/pages/ExperimentDetails/TrialsComparisonModal.module.scss [hunk 1]
```diff
@@ -19,7 +19,9 @@
     }
     th,
     td {
+      contain: paint;
       padding: 8px;
+      text-overflow: ellipsis;
```
Comment (EmilyBonar) [inline webui/react/src/pages/ExperimentDetails/TrialsComparisonModal.module.scss:24]: nit: i think these changes are unnecessary after adding typography

**Review by EmilyBonar — APPROVED** [review]
LGTM

**Pull request merged**